src/commands/clan/clan.ts

```diff
@@ -13,34 +18,34 @@ const DREAD_BOSS_MAPPINGS = new Map([
   ["ghost", "Mayor Ghost"],
   ["vampire", "Drunkula"],
   ["skeleton", "Unkillable Skeleton"],
-  ["xwerewolf", "~~Air Wolf~~"],
-  ["xbugbear", "~~Falls-From-Sky~~"],
-  ["xzombie", "~~Zombie HOA~~"],
-  ["xghost", "~~Mayor Ghost~~"],
-  ["xvampire", "~~Drunkula~~"],
-  ["xskeleton", "~~Unkillable Skeleton~~"],
+  ["xwerewolf", strikethrough("Air Wolf")],
```

Comment:
but pagliacci, what if i like squiggly lines

Reply:
(NOTE: strikethrough( scans better it is fine i just like squiggles)

Reply:
The squiggles are fine yeah but I like the formatters